Sims1WidescreenPatcher.Core/Tabs/ExtrasTabViewModel.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System.Reactive;
+using System.Reactive.Linq;
+using ReactiveUI;
+using Sims1WidescreenPatcher.Core.Events;
+using Sims1WidescreenPatcher.Core.Factories;
+using Sims1WidescreenPatcher.Core.Models;
+using Sims1WidescreenPatcher.Core.Services;
+using Sims1WidescreenPatcher.Core.Services.Interfaces;
+using Sims1WidescreenPatcher.Core.ViewModels;
+
+namespace Sims1WidescreenPatcher.Core.Tabs;
+
+public class ExtrasTabViewModel : ViewModelBase, IExtrasTabViewModel
+{
+    private readonly ICheatsService _cheatsService;
+    private readonly ObservableAsPropertyHelper<bool> _applyBtnVisible;
+    private CheckboxSelectionSnapshot _previousSnapshot;
+    private IAppState AppState { get; }
+
+    public ExtrasTabViewModel(CheckboxViewModelFactory creator, ICheatsService cheatsService, IAppState appState,
+        IProgressService progressService)
+    {
+        _cheatsService = cheatsService;
+        AppState = appState;
+        UnlockCheatsViewModel = (CheckboxViewModel)creator.Create("Unlock Cheats");
+        UnlockCheatsViewModel.ToolTipText = "Unlock cheats that were disabled in the release build the game";
```

Comment:
```suggestion
        UnlockCheatsViewModel.ToolTipText = "Unlock cheats that were disabled in the release build of the game.";
```

Reply:
Thank you for the review and for sticking around :)